daemon/volumes_unix.go

```diff
@@ -316,7 +317,7 @@ func (daemon *Daemon) registerMountPoints(container *Container, hostConfig *runc
 			}
 
 			if len(cp.Source) == 0 {
-				v, err := daemon.createVolume(cp.Name, cp.Driver, nil)
+				v, err := daemon.createVolume(cp.Name, cp.Driver, nil, stringid.TruncateID(containerID))
```

Comment:
@calavera  I think we should store the container name, what do you think? I think the container name is more human readable.